### DISCUSSION TITLE
Allow models to be edited directly from the index listing

### DISCUSSION THREAD

**Discussion by tabias** [wagtail/wagtail#11377] [Ideas]
I'm migrating away from django-cms and am using a lot of list_editable admin content (originally from django itself).

Sadly I can't seem to reproduce this in wagtail.

If someone could put me on the right way to implement this I could generate a pull request as such.

**Comment by gasman** [comment]
Unfortunately I suspect this will be a very substantial task - `list_editable` is a feature of Django's admin module, which django-cms is built on top of, but Wagtail isn't. You'd need to investigate how it's implemented in [`django.contrib.admin`](https://github.com/django/django/tree/master/django/contrib/admin), and see how to re-implement the same thing in Wagtail's listing views (`wagtail.wagtailadmin.views.pages`, `wagtail.wagtailsnippets.views` and various others).

You may well find that the concept doesn't translate directly to how Wagtail works - for example, Wagtail allows a user to have edit permissions over specific subsections of the page tree, so it's possible that a page listing could contain some entries that are editable and some that aren't. Given the scope of the change, I'd suggest writing up a detailed specification as an RFC: https://github.com/wagtail/rfcs

**Comment by tabias** [comment]
ok, I'll put it on my todo list for rfcs.  Tnx for the quick response.

**Comment by thibaudcolas** [comment]
I’m in agreement with @gasman on this, so will convert this to a Discussion now. It’d be an interesting feature for sure, but would require enough thought to warrant an RFC (or at the very least, way more discussion).

**Reply by thibaudcolas** [reply]
cc @mtrythall as our resident django-cms expert – would you say it’s a feature Wagtail would benefit from?

**Comment by Atem18** [comment]
Hi, I am migrating away a website from Symphony and their easyadmin panel seems to have the same functionally where you can change the status of a record or access a foreignkey record from the list of all the records in the admin.

Wagtail would really benefit from it IMO as this is a pain for editors to have to go to each record to just toggle a simple button.
